src/GUI/RevEng.Shared/TableInformationModel.cs

```diff
@@ -47,20 +48,37 @@ public bool HasPrimaryKey
             }
         }
 
+        /// <summary>
+        /// Gets or sets whether the table or view is keyless - always false for EF Core 2.0.
+        /// </summary>
```

Comment:
Maybe add an `<example>` section below `<summary>` explaining the difference between `HasPrimaryKey` and `IsKeyless`.